README.md

```diff
@@ -112,490 +76,16 @@ Secures your IaC files. Critical to keeping your cloud deployment safe and secur
-
-    - On macOS and Linux:
-
-        ```sh
-        source <venv-dir>/bin/activate
-
-        # For example:
-        source .env/bin/activate
-        ```
-
-    - On Windows:
-
-        ```powershell
-        .\<venv-dir>\Scripts\activate
-
-        # For example:
-        .\env\Scripts\activate
-        ```
-
-5. In the same terminal, install your python project and dependencies according to your project specifications.
-
-### .NET Projects
-
-For .NET projects which use NuGet packages as dependencies, the extension displays the NuGet dependencies tree, together with the information for each dependency.
-Behind the scenes, the extension builds the NuGet dependencies tree using the [NuGet deps tree](https://github.com/jfrog/nuget-deps-tree) npm package.
-
-Important:
-
--   Does your project define its NuGet dependencies using a _packages.config_ file? If so, then please make sure the `nuget` CLI is installed on your local machine and that it is in your system PATH. The extension uses the `nuget` CLI to find the location of the NuGet packages on the local file-system.
--   The project must be restored using `nuget restore` or `dotnet restore` prior to scanning. After this action, you should click on the Refresh ![Refresh](resources/readme/refresh.png) button, for the tree view to be refreshed and updated.
-
-## Troubleshooting
-
-Change the log level to 'debug', 'info', 'warn', or 'err' in the [Extension Settings](#extension-settings).
-
-View the extension log:
-![Logs](resources/readme/gifs/logs.gif)
-
-## License
-
-The extension is licensed under [Apache License 2.0](LICENSE).
-
-## Building the Sources
-
--   npm 7 and above
--   JFrog CLI's `jf` executable - required for tests
-
-To build the extension from sources, please follow these steps:
-
-1. Clone the code from Github.
-2. Update submodules:
-
-```bash
-git submodule init
-git submodule update
-```
-
-3. Build and create the VS-Code extension vsix file by running the following npm command:
-
-```bash
-npm i
-npm run package
-```
-
-After the build finishes, you'll find the vsix file in the _jfrog-vscode-extension_ directory.
-The vsix file can be loaded into VS-Code
-
-###  Test Instructions:
-* create a remote repository named `releases-proxy` designed to function as a  [proxy jfrog releases](external-resource-repository).
-* Set the environment variables JFROG_IDE_PLATFORM_URL and JFROG_IDE_ACCESS_TOKEN to establish a connection with a valid JFrog instance.
-
-To run the test, execute the following command:
-
-```bash
-npm t
-```
-
-## Code Contributions
+## 🔥 Reporting Issues
+Please help us improve by [reporting issues](https://github.com/jfrog/jfrog-vscode-extension/issues) you encounter.
 
+## 💻 Code Contributions
 We welcome community contribution through pull requests.
 
 ### Guidelines
-
 -   Before creating your first pull request, please join our contributors community by signing [JFrog's CLA](https://secure.echosign.com/public/hostedForm?formid=5IYKLZ2RXB543N).
```

Comment:
This link appears to be broken: 
https://secure.echosign.com/public/hostedForm?formid=5IYKLZ2RXB543N
![image](https://github.com/jfrog/jfrog-vscode-extension/assets/9606235/c555c5b3-01b0-45ae-b21f-72eea91f3668)